Jakefile

```diff
@@ -303,19 +296,10 @@ compileFile(processDiagnosticMessagesJs,
 // The generated diagnostics map; built for the compiler and for the 'generate-diagnostics' task
 file(diagnosticInfoMapTs, [processDiagnosticMessagesJs, diagnosticMessagesJson], function () {
     var cmd = "node " + processDiagnosticMessagesJs + " "  + diagnosticMessagesJson;
-    console.log(cmd);
-    var ex = jake.createExec([cmd]);
-    // Add listeners for output and error
-    ex.addListener("stdout", function(output) {
-        process.stdout.write(output);
-    });
-    ex.addListener("stderr", function(error) {
-        process.stderr.write(error);
-    });
-    ex.addListener("cmdEnd", function() {
+
+    exec(cmd, function() {

```

Comment:
Why are these `function() { complete(); }` callbacks preferable to the old behavior?